lib/nostrum/consumer.ex

```diff
@@ -74,7 +74,7 @@ defmodule Nostrum.Consumer do
   @typedoc """
   The two different types of channels - Guild channels and DM channels.
```

Comment:
I think this should be changed to be more accurate, or we could remove this type from this module and change the types below to point towards the Channel struct type.